### PR TITLE
fix(ui5-notification-list): improved accessibility

### PR DESCRIPTION
**Pull request by LidiyaGeorgieva**
- read state is announced after the title (before it was at the end of the notification)
- "notification" word is removed from the description
- "important" should be announced before the title and the status text, since it is valuable information for the user and should be in the beginning
- status icon in the title should have tooltip
- there should be "loading" announcement (after the title) for items, which are in loading state
- ACC attributes are added for "More"/"Less" link, following the example of (openui5) sap.m.ExpandableText
- tooltip for the expand/collapse arrow in the group header is moved to the whole header
- tooltip for the group header is changed to static "Expand/Collapse" (doesn't change depending on the expand state of the group), following the example of ui5-panel component
- samples in website playground fixed (there was a bug, that the popover doesn't open in the notifications-in-shellbar sample)
- tests added

### CODE REVIEW

File: packages/fiori/src/i18n/messagebundle.properties [hunk 1]
```diff
@@ -44,9 +41,18 @@ NOTIFICATION_LIST_ITEM_SHOW_LESS=Less
 #XBUT: Tooltip text for 'Overflow' button in the NotificationListItem, that opens a menu with actions
 NOTIFICATION_LIST_ITEM_MENU_BTN_TITLE=Actions
 
+#XBUT: Label text for 'More' lin in the NotificationListItem, that shows the full text, if truncated
```
Comment (BorisDafov) [inline packages/fiori/src/i18n/messagebundle.properties:44]: Label text for 'More' link

File: packages/fiori/src/i18n/messagebundle.properties [hunk 2]
```diff
@@ -44,9 +41,18 @@ NOTIFICATION_LIST_ITEM_SHOW_LESS=Less
...
+NOTIFICATION_LIST_ITEM_MORE_LINK_LABEL_FULL=More button. Show the full texts
```
Comment (BorisDafov) [inline packages/fiori/src/i18n/messagebundle.properties:45]: the button shows only one text for the respective item. I suggest:
...Show the full text

File: packages/fiori/src/i18n/messagebundle.properties [hunk 3]
```diff
@@ -44,9 +41,18 @@ NOTIFICATION_LIST_ITEM_SHOW_LESS=Less
...
+
+#XBUT: Label text for 'More' lin in the NotificationListItem, that hides the truncated text
```
Comment (BorisDafov) [inline packages/fiori/src/i18n/messagebundle.properties:47]: I suggest:
Label text for 'More' link in the NotificationListItem, that truncates the text

File: packages/fiori/src/i18n/messagebundle.properties [hunk 4]
```diff
@@ -44,9 +41,18 @@ NOTIFICATION_LIST_ITEM_SHOW_LESS=Less
...
+NOTIFICATION_LIST_ITEM_MORE_LINK_LABEL_TRUNCATE=Less button. Show the texts with truncation
```
Comment (BorisDafov) [inline packages/fiori/src/i18n/messagebundle.properties:48]: one text:
Show the text with truncation

**Review by BorisDafov — CHANGES_REQUESTED** [review]

File: packages/fiori/src/i18n/messagebundle.properties (hunk 2 above)
Comment (LidiyaGeorgieva) [reply packages/fiori/src/i18n/messagebundle.properties:45]: The "button" show the full text of the title and full text of the description, so they are two texts
image

File: packages/fiori/src/i18n/messagebundle.properties (hunk 4 above)
Comment (LidiyaGeorgieva) [reply packages/fiori/src/i18n/messagebundle.properties:48]: as above

File: packages/fiori/src/NotificationListGroupItem.hbs [hunk 5]
```diff
@@ -13,11 +13,12 @@
...
 		role="button"
 		aria-expanded="{{_ariaExpanded}}"
-		aria-controls="{{_id}}-notificationsList">
+		aria-controls="{{_id}}-notificationsList"
+		title="{{toggleIconAccessibleName}}">
 		<ui5-icon
 			name={{groupCollapsedIcon}}
-			show-tooltip="true"
-			accessible-name={{toggleIconAccessibleName}}
+			{{!-- show-tooltip="true"
```
Comment (TeodorTaushanov) [inline packages/fiori/src/NotificationListGroupItem.hbs:20]: remove it

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]

**Review by TeodorTaushanov — APPROVED** [review]

**Pull request merged**